spec/openapi.yaml

```
@@ -120,6 +120,39 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /tag/metadata/delete:
+    post:
+      operationId: tag-metadata-delete
+      summary: Permanently deletes the specified metadata from an existing tag.
+      description: >
+        Provide the metadata names that you want to delete.
+      tags:
+        - Tags
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TagMetadataDeleteRequest'
+      responses:
+        '200':
+          description: Tag metadata successfully deleted.
```

```suggestion
          description: Tag metadata successfully deleted. This operation is idempotent, and will also return HTTP code 200 if some or all of the tag metadata names were not found.
```

If you agree with [this suggestion](https://github.com/wisetime-io/connect-api-spec/pull/29/files#r396850452), then perhaps something like the above makes sense.

---

Removed 204 from spec.